hbase-client/src/main/java/org/apache/hadoop/hbase/ipc/HBaseRpcControllerImpl.java

```diff
@@ -101,8 +102,12 @@ public void setPriority(int priority) {
 
   @Override
   public void setPriority(final TableName tn) {
-    setPriority(
-      tn != null && tn.isSystemTable() ? HConstants.SYSTEMTABLE_QOS : HConstants.NORMAL_QOS);
+    int priority = HConstants.NORMAL_QOS;
+    if (tn != null && tn.isSystemTable()
+        && !tn.equals(SlowLogTableAccessor.SLOW_LOG_TABLE_NAME)) {
```

Comment:
May be move this to a util method some where. Like isMeta that we have. We can say as isLogTable()?

Reply:
Should we set the priority on the write req directly (Put) rather than having extra check like this at different places?  That will work right?

Reply:
Oh, okk sure let me update priority with Put request. So in that case, we don't need this change and even the above class change `tableName.isSystemTable() && !tableName.equals(SlowLogTableAccessor.SLOW_LOG_TABLE_NAME)`.
Thanks @ramkrish86 @anoopsjohn